### DISCUSSION TITLE
Plater Updates If Model Changes

### DISCUSSION THREAD

**Discussion by 24c** [supermerill/SuperSlicer#1849] [General]
Hi

I've noticed that Cura 4.11 updates or tells you a file on the build plate has updated elsewhere, and asks if you want to update the model, but SuperSlicer doesn't. I'm not sure if this is a macOS thing or whether there isn't this functionality.

I must admit in Cura with another printer, it is a useful feature, especially when you are refining a CAD model, and have the slicer open making parts. Is this functionality available in SuperSlicer? and is it anything anyone else thinks is useful?

Mike

**Comment by supermerill** [comment]
You can reload the part if you modified it (right clic -> reload from disk). I think prusa is working on enhancing that reload so it can also be activated on modified objects.
There is no check to tell the user if the current loaded object original file has been modified.
Cura does it only when loading the project or is there a hidden process that check the filesystem every (tens) seconds?

**Reply by 24c** [reply]
OK, no problem, but thank you for a quick response. I will wait and see what develops. :)
